src/menu/menu.c

```diff
@@ -341,7 +347,11 @@ static bool
 nodename_supports_cdata(char *nodename)
 {
 	return match_glob("command.action.item.*menu.openbox_menu", nodename)
-		|| match_glob("execute.action.item.*menu.openbox_menu", nodename);
+		|| match_glob("execute.action.item.*menu.openbox_menu", nodename)
+		|| match_glob("command.action.item.openbox_pipe_menu", nodename)
+		|| match_glob("execute.action.item.openbox_pipe_menu", nodename)
+		|| match_glob("command.action.item.*menu.openbox_pipe_menu", nodename)
+		|| match_glob("execute.action.item.*menu.openbox_pipe_menu", nodename);
```

Comment:
Could we use `!strncmp(nodename, "execute.action", strlen("execute.action"))` here and only check if nodename starts with `execute.action` and `command.action`?

Reply:
Very good point. Done.